Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:testing
+
+# Install dependencies
+RUN \
+  apt-get update && \
+  apt-get install -y git python3-pip python3-venv && \
+  pip3 install keyring && \
+  git clone git://github.com/psf/requests.git && \
```

Comment:
why do you need to `git clone` this? why not just `pip3 install requests`?

Reply:
I don't know :) I must have misread the installation instructions.